src/Servers.jl

```diff
@@ -291,6 +291,7 @@ function listen(f::Function,
     tcpserver = Base.listen(getaddrinfo(host), port)
 
     tcpref[] = tcpserver
+    conn = 1
```

Comment:
Maybe call this `count`?

Reply:
But, it seems that instead of passing a connection count down through the stack just for log identification, maybe it's better to use the new `with_logger` feature to keep track of logs per connection:

https://github.com/JuliaLang/julia/pull/26005/files#diff-3e96068c2d524c60e771965c63dca20aR125

https://github.com/JuliaLang/julia/issues/25109#issuecomment-363608019